.github/workflows/ghpages.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           cache: npm
           cache-dependency-path: '**/package-lock.json'
       - working-directory: page
-        run: npm install -D --force
```

Comment:
Maybe we need to kept this tag

Reply:
Would you mind to enlighten me about this?

Reply:
We got a problem when we update the previous version, the tag --force was mandatory

Reply:
I have fixed that problem